contexts/did-v1.jsonld

```diff
@@ -50,17 +50,11 @@
       "@id": "dc:created",
       "@type": "xsd:dateTime"
     },
-    "blockchainAccountId": "sec:blockchainAccountId",
     "keyAgreement": {
       "@id": "sec:keyAgreementMethod",
       "@type": "@id",
       "@container": "@set"
     },
-    "publicKey": {
```

Comment:
if this term is removed, all examples will need to include the sec context right?

Reply:
All examples that use `publicKey` would, yes.

Again, I'm totally fine to keep this in if it's included on the "special" list, and we can define what that "special" list is and how things get added to it.

Reply:
I think this change would come as a shock to most JSON developers.

Reply:
Why do JSON developers care?? They don't need `@context` at all!

Reply:
I would prefer not to invite incompatibility any more than we already have. My point is that this term is valid in JSON, and you are making it invalid in JSON-LD... without extension... IMO, not a helpful change :)

Reply:
I think noting that its only defined in security-vocab and warning that its deprecated is as far as we can go, without needlessly breaking a bunch of JSON that was valid for both JSON-LD and JSON before, and wouldn't be after this is merged.

Reply:
Do you mean the case where a technically JSON-LD implementation just glues the did/v1 context to the top but otherwise treats it as plain JSON, and uses `publicKey`, and is not at all motivated to correct their now-faulty JSON-LD use because what they really want is plain JSON and to not have to care about this stuff, and are just using the `@context` as a courtesy but if we try to make them add another line they're going to throw the toys out the pram?

That's fair enough. Valid case I think :)

BUT if JSON-LD implementers are completely prevented from making breaking changes to *unstable* (and stated as such) v1 context because of plain JSON developers, that's kind of rubbish.

Reply:
@rhiaro I 100% agree.... and would advise us to never publish an "unstable context" with a URL like "https://example.com/v1" ever again, because people will treat it as immutable.

its why we created https://w3id.org/security/v3-unstable and I push breaking change to it daily :)